libs/@blockprotocol/graph/src/graph-module.json

```
@@ -58,17 +52,33 @@
       "sentOnInitialization": false,
       "data": {
         "type": "object",
-        "properties": {
-          "entityId": {
-            "description": "The entityId of the entity to update",
-            "type": "string"
+        "allOf": [
+          {
+            "properties": {
+              "entityId": {
+                "description": "The entityId of the entity to update",
+                "$ref": "https://blockprotocol.org/types/modules/graph/0.3/schema/entity-id"
+              },
+              "entityTypeId": {
+                "description": "The entityTypeId of the type of the entity to update",
+                "$ref": "https://blockprotocol.org/types/modules/graph/0.3/schema/versioned-url"
+              },
+              "properties": {
+                "description": "The properties of the entity to update",
```

Let's go back to this

```suggestion
                "description": "The new properties object to assign to the entity",
```

---

It seems clearer to me that it's overwriting the properties object